### PR TITLE
Added Q2 2022 roadmap items

### PR DESCRIPTION
**Pull request by chrishumanitec**

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
+Humanitec already provides a range of core resource types such as [`dns`](https://docs.humanitec.com/integrations/resource-types/dns), [`postgres`](https://docs.humanitec.com/integrations/resource-types/postgres) and [`volume`](https://docs.humanitec.com/integrations/resource-types/volume). Since the introduction of Resource References (previously known as Dependent Resources) a lot more can be achieved with the resources system. We will be introducing a number of new types including `tsl-cert` for modelling TLS Certificates, `k8s-namespace` to make it easier to programmatically defined the name of namespaces and additional types to model dependencies on services in other applications or hosted in other platforms.
```
Comment (sujaya-sys) [inline docs/roadmap.md:9]: typo: "to programmatically define"

File: docs/roadmap.md [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
 
-### App Details Page UI Update
+### Virtual Drivers
 
-Following several months of user testing and customer feedback we are planning on introducing a number of changes to improve the usability, predictability and scalability of our UI. The focus lies on the App Details Page, the control centre of an App. We want to address issues related to interaction design, information hierarchy, status communication as well as aesthetics while touching on UI patterns which are likely to change the look and feel of the Humanitec UI as a whole.
+The introduction of generic drivers such as the [`humanitec/template`](https://docs.humanitec.com/integrations/resource-drivers/template) driver add a lot of power to the resource system. Unfortunately, using these drivers multiple times requires the driver inputs to be duplicated. This makes it hard to maintain and understand these [Resource Definitions](https://docs.humanitec.com/reference/concepts/resources/definitions). Virtual Drivers will provide a way of encapsulating and reusing the inputs of a driver across multiple [Resource Definitions](https://docs.humanitec.com/reference/concepts/resources/definitions).
```
Comment (sujaya-sys) [inline docs/roadmap.md:13]: "resource system" vs. "resources system" (see below "More flexible Resource Types)

File: docs/roadmap.md (hunk 1 above)
Comment (Nilsty) [reply docs/roadmap.md:9]: there is another typo 
it reads: `tsl-cert`
it should read: `tls-cert`

File: docs/roadmap.md (hunk 2 above)
Comment (chrishumanitec) [reply docs/roadmap.md:13]: This is not actually a typo!

File: docs/roadmap.md (hunk 1 above)
Comment (chrishumanitec) [reply docs/roadmap.md:9]: Will fix them both.

**Review by christophcrichter — APPROVED** [review]
LGTM

**Pull request merged**